src/DotNetWorker.Core/Converters/Converter/EnableConvertersFallbackAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.Functions.Worker.Converters
+{
+    /// <summary>
+    /// An attribute that specifies if Converters fallback is enabled
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class)]
```

Comment:
I believe we want this to be usable anywhere `InputConverterAttribute` is usable (so parameters, types, etc). Although keep `AllowMultiple = false`

Reply:
Done